src/Makefile

```diff
@@ -276,3 +276,6 @@ deploy-helm:
 
 	HELM_RELEASE=$(HELM_RELEASE) \
 	./newrelic_deployment.sh
+
+	HELM_RELEASE=$(HELM_RELEASE) \
+	./run_bash_script_in_php_pod.sh ls.sh
```

Comment:
+1 I like the abstraction here.
Perhaps can be extended to a directory named, for example  `post_deploy_tasks`, and the script executes each script in the directory - not a blocker to this but a possible enhancement later on.
Similar to the way each container sources the folder `/app/env` on startup: https://github.com/greenpeace/planet4-docker/blob/master/src/planet-4-151612/ubuntu/bin/entrypoint.sh#L6-L10

Reply:
That is similar to (but better than) my proposal for the next version: 
My proposal: 
1) Use composer.json to add post-deploy-one-pod-script relevant to all p4 installations
2) Use composer.json to add post-deploy-every-pod-script relevant to all p4 installations
3) Use composer-local.json to add post-deploy-one-pod-script relevant to one p4 installation
4) Use composer.json to add post-deploy-one-pod-script relevant to one p4 installation

Make one script in the makeFile that would run (if they exist), the one-pod scripts in one pod after deploy and the all-pods scripts in every pod after deploy.

I am not sure that we would even need an all-pods script ever, I was just trying to anticipate (no need to deploy it now). 

We will go with your proposal of course, but as a next step.